db/migrate/20151107030456_add_attachment_avatar_to_people.rb

```diff
@@ -0,0 +1,11 @@
+class AddAttachmentAvatarToPeople < ActiveRecord::Migration
+  def self.up
+    change_table :people do |t|
```

Comment:
What happens for all the existing people before pictures were added? Do we need to drop any existing people?